schema/invitation.js

```diff
@@ -4,6 +4,24 @@ class Invitation extends BaseModel {
   static get tableName() {
     return 'invitations';
   }
+
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: ['role', 'establishmentId', 'profileId'],
+      properties: {
+        token: { type: 'string' },
+        role: {
+          type: 'string',
+          enum: ['basic', 'read', 'admin']
```

Comment:
This is expected to be `readonly` rather than `read` in the permissions resolver.  Unless you know of somewhere that's expecting it to be `read`?

Reply:
I got all the enums from the migrations:
https://github.com/UKHomeOffice/asl-schema/blob/master/migrations/20180807084155_added_invitaions_table.js#L6

Reply:
Looks like I need to change the permissions then...

Reply:
@lennym is permissions resolver a _thing_? Asking for a friend

Reply:
@joefitter I meant asl-permissions.